### DISCUSSION TITLE
docsify with ui frameworks for theme?

### DISCUSSION THREAD

**Discussion by ghost** [docsifyjs/docsify#1941] [Q&A] [answered]
Hi everyone.

#### Idea
I would like to develop more themes in docsify with ui-frameworks like: Boostrap, Tabler-ui, Skeleton, Bulma, Pure.css, Groundwork, Cardinal, PowertoCSS, Mueller, Bootflat, Flowbite, [fish-ui](https://github.com/myliang/fish-ui), [uikit/uikit](https://github.com/uikit/uikit)

#### flowbite in  docsify
```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/flowbite.css" />
```

#### boostrap in  docsify
```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/bootstrap.css" />
```

#### tabler-ui in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/tabler-ui.css" />
```

#### skeleton in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/skeleton.css" />
```

#### bulma in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/bulma.css" />
```

#### pure-css in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/pure-css.css" />
```

#### groundwork in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/groundwork-css.css" />
```

#### cardinal in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/cardinal.css" />
```

#### powertoCSS in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/powertoCSS.css" />
```

#### mueller in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/mueller.css" />
```

#### bootflat in docsify

```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/bootflat.css" />
```

#### fish-ui in docsify
```html
<link rel="stylesheet" href="//cdn.jsdelivr.net/npm/docsify@4/themes/fish-ui.css" />
```

#### question/feedback
What do you think of this idea?

**Comment by akbatra567** [comment, marked as answer]
Sounds like a great idea, it adds the flexibility for theme choosing for user as well as learning opportunities for the developers.

**Reply by ghost** [reply]
Hi akbatra567.

> Sounds like a great idea, it adds the flexibility for theme choosing for user as well as learning opportunities for the developers.

thank you for feedback.
